pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -887,4 +887,66 @@ public void testQueryValidation() {
       Assert.assertTrue(e.getMessage().contains("is not allowed in GROUP BY clause."));
     }
   }
+
+  @Test
+  public void testAliasQuery() {
```

Comment:
For completeness, can we also add a test case where we have a mix of expressions and for some we use AS, and for others we don't.

Reply:
Updated the tests